src/components/components/HeroSection/HeroSection.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./HeroSection.scss";
+
+function HeroSection() {
+	return (
+		<div className="HeroSection">
+			<div className="HeroSection-main">
+				<h1 className="HeroSection-main__title">
+					Uniting{" "}
+					<span className="HeroSection-main__title--circled">skill</span> with
+					opportunity
+				</h1>
+				<p className="HeroSection-main__text">
+					SkillChain connects job seekers with top employers to find the perfect
+					match for job vacancies.
+				</p>
+				<a href="#" target="_blank">
+					<button className="HeroSection-main__button">View Jobs</button>
+				</a>
+			</div>
+			<div className="HeroSection__image"></div>
+		</div>
+	);
+}
```

Comment:
The spacing looks weird to me. It looks as if there are always 1 tab more than there should be

Reply:
something wrong with the alignment of the image in other screen sizes except 1440*924. i think the image should always at the bottom

Reply:
valid point, thank you! I have corrected the styles to make it responsive. Should be fixed now (see the latest commit)